src/Frontend/Pages/CreateLobby.tsx

```diff
@@ -150,16 +198,180 @@ export function CreateLobby({ match }: RouteComponentProps<{ contract: string }>
     }
   }, [contract]);
 
+  async function createAndRevealPlanets(initializers: LobbyInitializers) {
```

Comment:
can we move all of these functions to `Utils`? Or somewhere that is abstracted away from the key details of how a lobby is created?